src/lib/tests/MockRelayRenderer.tsx

```diff
@@ -2,15 +2,34 @@ import { IMocks } from "graphql-tools/dist/Interfaces"
 import React from "react"
 import { QueryRenderer, RelayContainer } from "react-relay"
 import { Environment, GraphQLTaggedNode, OperationBase, OperationDefaults, RecordSource, Store } from "relay-runtime"
-import { ContextProvider } from "../utils/Context"
+import { ContextConsumer, ContextProvider } from "../utils/Context"
 import renderWithLoadProgress from "../utils/renderWithLoadProgress"
-import { createMockNetworkLayer } from "./createMockNetworkLayer"
+import { createMockNetworkLayer, createMockNetworkLayer2 } from "./createMockNetworkLayer"
```

Comment:
`createMockNetworkLayer2`?

Reply:
I just followed the naming from the reaction code but yeah I agree it's a lil cryptic